Lib/test/test_typing.py

```diff
@@ -7515,7 +7499,6 @@ class Y(Generic[P, T]):
         self.assertEqual(B.__args__, ((int, str,), Tuple[bytes, float]))
 
     def test_var_substitution(self):
-        T = TypeVar("T")
```

Comment:
`T` here is currently an unused variable. `TypeVar.__typing_subst__` is tested more fully elsewhere:

https://github.com/python/cpython/blob/7894bbe94ba319eb650f383cb5196424c77b2cfd/Lib/test/test_typing.py#L461-L472